src/App.js

```diff
@@ -202,7 +202,8 @@ class App extends React.Component {
         this.setState({ wrapper })
       })
       .catch(err => {
-        console.error('Wrapper init error:', err)
+        console.error('Error:', err)
+        this.setState({ appsLoading: false })
```

Comment:
Should this also set the error into `fatalError`?

Reply:
https://github.com/aragon/aragon/pull/365 should provide a better experience in that case.